jscomp/ext/ext_bytes.ml

```diff
@@ -55,6 +58,10 @@ let escaped s =
       begin match Bytes.unsafe_get s i with
       | ('"' | '\\') as c ->
           Bytes.unsafe_set s' !n '\\'; incr n; Bytes.unsafe_set s' !n c
+#if BS_NATIVE then
+      | '$' ->
+           Bytes.unsafe_set s' !n '$'; incr n; Bytes.unsafe_set s' !n '$'
+#end
       | '\n' ->
```

Comment:
where is this function used in bs_native? The escaping rules does not seem correct to me

Reply:
If you read the docs of {String.ecaped}, it escape the special character following ocaml lexical convention which is not relevant to windows path convention, I guess you need a special function for windows path escaping